doc/api/n-api.md

```diff
@@ -1331,6 +1331,38 @@ JavaScript TypedArray Objects are described in
 [Section 22.2](https://tc39.github.io/ecma262/#sec-typedarray-objects)
 of the ECMAScript Language Specification.
 
+
+#### *napi_create_dataview*
+<!-- YAML
+added: v8.0.0
+-->
+```C
+napi_status napi_create_dataview(napi_env env,
+                                   size_t length,
+                                   napi_value arraybuffer,
+                                   size_t byte_offset,
+                                   napi_value* result)
```

Comment:
Can you align the arguments vertically (i.e. `size_t length` directly below `napi_env env` etc.?)